Alignment/OfflineValidation/python/TkAlAllInOneTool/geometryComparison.py

```diff
@@ -252,9 +252,6 @@ def createScript(self, path):
                 resultingFile = os.path.expandvars( resultingFile )
                 resultingFile = os.path.abspath( resultingFile )
                 resultingFile = "root://eoscms//eos/cms" + resultingFile   #needs to be AFTER abspath so that it doesn't eat the //
-                repMap["runComparisonScripts"] += \
-                    ("xrdcp -f OUTPUT_comparison.root %s\n"
-                     %resultingFile)
```

Comment:
The effect is basically to remove a bash error because OUTPUT_comparison.root does not exist.

I think this line was supposed to copy the output of [makeArrowPlots("comparison.root", "...")](https://github.com/cms-sw/cmssw/blob/1348d5a/Alignment/OfflineValidation/scripts/makeArrowPlots.C#L209)


But actually the first argument to makeArrowPlots is [something else](https://github.com/cms-sw/cmssw/blob/1348d5a/Alignment/OfflineValidation/python/TkAlAllInOneTool/geometryComparison.py#L233), so this file is never created and it just gives a warning.

The actual output file contains .oO[name]Oo. so it gets copied in [this step](https://github.com/cms-sw/cmssw/blob/1348d5a/Alignment/OfflineValidation/python/TkAlAllInOneTool/configTemplates.py#L104).